cjwilburn/dank-memes
# Add "meme roulette"

# Description
This adds a "meme roulette" mode wherein a user will be asked to provide 1 or 2 captions at the prompt. The caption(s) will be applied to the first and second zones for a random meme, and resulting meme created and opened to browser.

This expands on the initial "get meme" functionality and adds (for now) a throw-away username and password. **Note:** this username and password should probably be replaced at some point, or ideally provided by an end-user, or sheltered away from public repository.

## Steps to Test
1. Type `node roulette.js`
2. Provide a caption and hit enter. Provide a second caption and hit enter. View the resulting creation.
3. Provide only one caption and hit enter to skip second one. View the resulting creation.
4. Provide no initial caption. Should fail.


## Screenshots
http://i.imgflip.com/2jh6u7.jpg
http://i.imgflip.com/2jh6xn.jpg
http://i.imgflip.com/2jh706.jpg (1 caption only)
Example dank generated combination of user-provided captions and random imagery.


## Deploy Notes



## Checklist
